sympy/stats/error_prop.py

```diff
@@ -0,0 +1,74 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
```

Comment:
Haha. Anthony [wrote his own editor](https://github.com/scopatz/xo). But I bet he could easily integrate pyflakes with it if he wanted.